api/RTCSessionDescription.json

```diff
@@ -0,0 +1,151 @@
+{
+  "api": {
+    "RTCSessionDescription": {
```

Comment:
Constructor + toJSON methods are missing in the subfeature set here, please add them